cmd/packageExport.go

```diff
@@ -60,13 +60,19 @@ func newPackageExportCommand() *packageExportCommand {
 				}
 			}
 
+			var exportDest = &destination
+			if stdout {
+				exportDest = nil
+			}
 			// Export the packages
 			for _, pkg := range packages {
-				exportedTo, err := session.packageService.ExportPackageToConfig(*pkg, ".")
-				if err != nil {
-					message.Warningf("failed to export package %s to %s, %v", pkg.Label, exportedTo, err)
-				} else {
-					message.Successf("successfully exported package %s to %s", pkg.Label, exportedTo)
+				exportedTo, err := exportPackage(pkg, exportDest)
+				if err != nil && exportedTo != nil {
+					message.Warningf("failed to export package %s to %s, %v", pkg.Label, *exportedTo, err)
+				} else if err != nil {
+					message.Warningf("failed to export package %s, %v", pkg.Label, err)
+				} else if exportedTo != nil {
+					message.Successf("successfully exported package %s to %s", pkg.Label, *exportedTo)
```

Comment:
I think I worked out a slight simplification of this section. I like your solution but I think it would be easier to read/understand if we'd instead directly pass the `toStdout` flag to the `exportPackage` function and define the export behavior from there.

```go
// Export the packages
for _, pkg := range packages {
	exportedTo, err := exportPackage(pkg, destination, toStdout)
	if err != nil {
		if exportedTo != "" {
			message.Warningf("failed to export package %s to %s, %v", pkg.Label, exportedTo, err)
		} else {
			message.Warningf("failed to export package %s, %v", pkg.Label, err)
		}
	} else {
		if !toStdout {
			message.Successf("successfully exported package %s to %s", pkg.Label, exportedTo)
		}
	}
}
```

Reply:
That's a good suggestion… I'll implement ``exportPackage`` according to your draft, then this section can be even further simplified.